### PR TITLE
update glossary section

### PR DESCRIPTION
**Pull request by drcandacemakedamoore**
We can survey the pilot participants for what additonal terms are important to them.

### CODE REVIEW

**Review by gcroci2 — CHANGES_REQUESTED** [review]
Thanks for having updated it. This will be very useful for the students! 

I think that this PR is blocked by #6 (PR #16), and eventually #4 - if we are going to develop it, which may be no more the case. Once all the lessons are merged into main, we should go through them and check whether all the mainly needed terms have been added here. I moved the issue (#14) to the In Progess column because of this reason.

**Review by gcroci2 — APPROVED** [review]

**Comment by gcroci2** [comment]
I added few terms to have a more complete (but still starting) glossary. If you approve them, feel free to merge the PR :) @drcandacemakedamoore

**Comment by drcandacemakedamoore** [comment]
I put a slight tweak onto the radiomics definition. Otherwise fine

**Pull request merged**
